Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG BASE_IMAGE_TAG="3.19.0-rc.835"
-FROM ghcr.io/xgovformbuilder/digital-form-builder-runner:$BASE_IMAGE_TAG as base
```

Comment:
Could this be in the `communitiesuk` org rather than your personal one?

Reply:
Yeah I agree

Reply:
Should we not just revert the changes to this file?